itftp/itftp.go

```diff
@@ -54,23 +54,23 @@ func (t Handler) HandleRead(filename string, rf io.ReaderFrom) error {
 
 	full := filename
 	filename = path.Base(filename)
-	l := t.Log.WithValues("event", "get", "filename", filename, "uri", full, "client", client)
+	log := t.Log.WithValues("event", "get", "filename", filename, "uri", full, "client", client)
 
 	// clients can send traceparent over TFTP by appending the traceparent string
 	// to the end of the filename they really want
 	longfile := filename // hang onto this to report in traces
 	ctx, shortfile, err := extractTraceparentFromFilename(context.Background(), filename)
 	if err != nil {
-		l.Error(err, "")
+		log.Error(err, "failed to extract traceparent from filename")
 	}
 	if shortfile != filename {
-		l = l.WithValues("filename", shortfile) // flip to the short filename in logs
-		l.Info("client requested filename '", filename, "' with a traceparent attached and has been shortened to '", shortfile, "'")
+		log = log.WithValues("shortfile", shortfile)
+		log.Info("traceparent found in filename", "filenameWithTraceparent", longfile)
 		filename = shortfile
 	}
 	// If a mac address is provided, log it. Mac address is optional.
 	mac, _ := net.ParseMAC(path.Dir(full))
```

Comment:
Same sort of general comments here, though it looks like `full` never starts with a slash?

If that is the case, I think it would be helpful to rename `mac` to `optionalMac`.

Reply:
I would rather see a comment here (and in the tftp code) that demonstrates the expected path.

In addition, renaming the variable to indicate that it is optional would that intent make that clear in multiple places (rather than a single comment that might get munged in the future).